homework4/main4.js

```diff
@@ -0,0 +1,149 @@
+// 1. Создать функцию multiply, которая будет принимать любое количество чисел и возвращать
+// их произведение: multiply(1,2,3) = 6 (1*2*3)
+
+function multiply() {
+    var s = 1;
+    if(arguments.length==0){
+        return 0;
+    }
+    for(var i=0; i<arguments.length; i++) {
+        s *= arguments[i];
+    }
+    return s
+}
+multiply(1,2,3)
+
+// 2. Создать функцию, которая принимает строку и возвращает строку-перевертыш:
+// reverseString(‘test’) // “tset”.
+function reverseString(str) {
+    var newString = "";
+    for (var i = str.length-1; i>=0; i--){
+        newString+= str[i];
+    }
+    return newString;
+}
+reverseString('test')
+
+// 3. Создать функцию, которая в качестве аргумента принимает строку из букв и возвращает строку,
+//     где каждый символ разделен пробелом и заменен на юникод-значение символа:
+//     getCodeStringFromText(‘hello’) // “104 101 108 108 111”
+
+function getCodeStringFrom(newString) {
+    var splits = newString.split("");
+    var strEnd = "";
+    for (var i = 0; i<splits.length; i++) {
+        let oneSum = splits[i];
+        let uniCode = oneSum.charCodeAt()+" ";
```

Comment:
`splits[i].charCodeAt()` would be better. There is no need for separate variable here